generated_files/cardinalities.tsv

```diff
@@ -1,4 +1,4 @@
-superstructure	substructure	cardinality
+superstructure	structure	cardinality

```

Comment:
I don't remember coding this change, but I also don't have strong feelings about if it is good or bad.